datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -1411,15 +1411,15 @@ fn to_substrait_type(dt: &DataType, nullable: bool) -> Result<substrait::proto::
                 parameter: Some(parameter::Parameter::DataType(substrait::proto::Type {
                     kind: Some(r#type::Kind::I32(r#type::I32 {
                         type_variation_reference: DEFAULT_TYPE_REF,
-                        nullability: default_nullability,
+                        nullability: r#type::Nullability::Unspecified as i32,
```

Comment:
We are not using the nullability of the parameter on the consumer side, so I temporarily change it to `Unspecified`.